### PR TITLE
ping pong read loop

### PR DESCRIPTION
**Pull request by faddat**
## Description

This PR fixes the disconnection issue seen with the websocket served on port 8546

Specifically, it ensures that the server will engage in keepalive messaging as part of the read loop. 

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: rpc/websockets.go [hunk 1]
```diff
@@ -33,6 +34,17 @@ import (
 	evmtypes "github.com/evmos/ethermint/x/evm/types"
 )
 
+const (
+	wsReadBuffer       = 1024
+	wsWriteBuffer      = 1024
+	wsPingInterval     = 60 * time.Second
+	wsPingWriteTimeout = 5 * time.Second
+	wsPongTimeout      = 30 * time.Second
+	wsMessageSizeLimit = 15 * 1024 * 1024
```
Comment (fedekunze) [inline rpc/websockets.go:43]: I feel this should be server config options instead of constants

Comment (faddat) [reply rpc/websockets.go:43]: ah, so!

I did it this way, because Ethereum does -- wdyt?

Comment (fedekunze) [reply rpc/websockets.go:43]: let's add them here: https://github.com/evmos/ethermint/tree/main/server/config

Comment (faddat) [reply rpc/websockets.go:43]: ah this is a very sensible place for that....

File: server/config/config.go [hunk 2]
```diff
@@ -74,6 +75,22 @@ type EVMConfig struct {
 	MaxTxGasWanted uint64 `mapstructure:"max-tx-gas-wanted"`
 }
 
+// WebSocketConfig defines the application configuration values for the web socket.
+type WebSocketConfig struct {
```
Comment (fedekunze) [inline server/config/config.go:79]: we should we move the `WsAddress` here (and rename it to `Address`)

**Review by fedekunze — APPROVED** [review]
LGTM, can you add a changelog entry @faddat

**Comment by fedekunze** [comment]
@faddat are you planning on updating this?

**Comment by faddat** [comment]
Hey,

**Pull request closed without merging**